requirements-win.txt

```diff
@@ -1,25 +1,55 @@
 #
-# This file is autogenerated by pip-compile
+# This file is autogenerated by pip-compile with python 3.7
 # To update, run:
 #
 #    pip-compile requirements-win.in
 #
+-e git+git://github.com/kollivier/pyeverywhere.git@dev#egg=pyeverywhere
```

Comment:
I know you guys have forked pyeverywhere, so you can switch back to your version if you merge the dev branch of PyEverywhere, which has the changes needed to notarization for it to work as a Github Action.